From baf3f5eede2697f20644ac917617b76456742945 Mon Sep 17 00:00:00 2001
From: Lukasz Czerwinski <l.czerwinski@samsung.com>
Date: Tue, 28 May 2013 09:03:43 +0200
Subject: [PATCH 0274/1302] tizen_defconfig update

This patch enables:
 - NFC subsystem
 - NCI protocol
 - NFC HCI implementation
 - SHDLC link layer for HCI NFC drivers
 - NFC PN544 driver support
 - GP2AP002A00F light/priximity sensor
 - LSM330DLC sensor IIO driver
 - buffer support within IIO

Signed-off-by: Lukasz Czerwinski <l.czerwinski@samsung.com>
Signed-off-by: Sylwester Nawrocki <s.nawrocki@samsung.com>
Signed-off-by: MyungJoo Ham <myungjoo.ham@samsung.com>
---
 arch/arm/configs/tizen_defconfig | 42 +++++++++++++++++++++++++++++++---------
 1 file changed, 33 insertions(+), 9 deletions(-)

diff --git a/arch/arm/configs/tizen_defconfig b/arch/arm/configs/tizen_defconfig
index 8b4dc03..0ab7ac0 100644
--- a/arch/arm/configs/tizen_defconfig
+++ b/arch/arm/configs/tizen_defconfig
@@ -1013,7 +1013,18 @@ CONFIG_RFKILL_LEDS=y
 # CONFIG_NET_9P is not set
 # CONFIG_CAIF is not set
 # CONFIG_CEPH_LIB is not set
-# CONFIG_NFC is not set
+CONFIG_NFC=y
+CONFIG_NFC_NCI=y
+CONFIG_NFC_HCI=y
+CONFIG_NFC_SHDLC=y
+
+#
+# Near Field Communication (NFC) devices
+#
+# CONFIG_NFC_PN533 is not set
+CONFIG_NFC_PN544=y
+CONFIG_NFC_PN544_I2C=y
+# CONFIG_NFC_MICROREAD is not set
 CONFIG_HAVE_BPF_JIT=y
 
 #
@@ -2499,10 +2510,6 @@ CONFIG_LEDS_TRIGGER_DEFAULT_ON=y
 #
 CONFIG_LEDS_TRIGGER_TRANSIENT=y
 # CONFIG_LEDS_TRIGGER_CAMERA is not set
-CONFIG_SENSORS_CORE=y
-CONFIG_SENSORS_AK8975_=y
-CONFIG_SENSORS_LSM330DLC=y
-CONFIG_SENSORS_LPS331=y
 # CONFIG_ACCESSIBILITY is not set
 # CONFIG_EDAC is not set
 CONFIG_RTC_LIB=y
@@ -2651,6 +2658,7 @@ CONFIG_STAGING=y
 # CONFIG_ADIS16220 is not set
 # CONFIG_ADIS16240 is not set
 # CONFIG_LIS3L02DQ is not set
+# CONFIG_SCA3000 is not set
 
 #
 # Analog to digital converters
@@ -2730,6 +2738,9 @@ CONFIG_STAGING=y
 #
 # Triggers - standalone
 #
+# CONFIG_IIO_PERIODIC_RTC_TRIGGER is not set
+# CONFIG_IIO_GPIO_TRIGGER is not set
+# CONFIG_IIO_SYSFS_TRIGGER is not set
 # CONFIG_IIO_SIMPLE_DUMMY is not set
 # CONFIG_ZSMALLOC is not set
 # CONFIG_USB_ENESTORAGE is not set
@@ -2796,14 +2807,20 @@ CONFIG_EXYNOS_IOMMU=y
 # CONFIG_EXTCON is not set
 # CONFIG_MEMORY is not set
 CONFIG_IIO=y
-# CONFIG_IIO_BUFFER is not set
-# CONFIG_IIO_TRIGGER is not set
+CONFIG_IIO_BUFFER=y
+# CONFIG_IIO_BUFFER_CB is not set
+CONFIG_IIO_KFIFO_BUF=y
+CONFIG_IIO_TRIGGERED_BUFFER=y
+CONFIG_IIO_TRIGGER=y
+CONFIG_IIO_CONSUMERS_PER_TRIGGER=2
 
 #
 # Accelerometers
 #
 # CONFIG_KXSD9 is not set
-# CONFIG_IIO_ST_ACCEL_3AXIS is not set
+CONFIG_IIO_ST_ACCEL_3AXIS=y
+CONFIG_IIO_ST_ACCEL_I2C_3AXIS=y
+CONFIG_IIO_ST_ACCEL_SPI_3AXIS=y
 
 #
 # Analog to digital converters
@@ -2827,10 +2844,14 @@ CONFIG_IIO=y
 #
 # Barometer sensors
 #
+CONFIG_SENSORS_LPS331=y
 
 #
 # Hid Sensor IIO Common
 #
+CONFIG_IIO_ST_SENSORS_I2C=y
+CONFIG_IIO_ST_SENSORS_SPI=y
+CONFIG_IIO_ST_SENSORS_CORE=y
 
 #
 # Digital to analog converters
@@ -2870,7 +2891,9 @@ CONFIG_IIO=y
 # CONFIG_ADIS16080 is not set
 # CONFIG_ADIS16136 is not set
 # CONFIG_ADXRS450 is not set
-# CONFIG_IIO_ST_GYRO_3AXIS is not set
+CONFIG_IIO_ST_GYRO_3AXIS=y
+CONFIG_IIO_ST_GYRO_I2C_3AXIS=y
+CONFIG_IIO_ST_GYRO_SPI_3AXIS=y
 # CONFIG_ITG3200 is not set
 
 #
@@ -2886,6 +2909,7 @@ CONFIG_IIO=y
 # CONFIG_ADJD_S311 is not set
 # CONFIG_SENSORS_TSL2563 is not set
 # CONFIG_VCNL4000 is not set
+CONFIG_GP2AP002A00F=y
 
 #
 # Magnetometer sensors
-- 
1.8.3.2

